cmd/kaf/kaf.go

```diff
@@ -29,6 +29,37 @@ func getConfig() (saramaConfig *sarama.Config) {
 		saramaConfig.Net.SASL.User = cluster.SASL.Username
 		saramaConfig.Net.SASL.Password = cluster.SASL.Password
 	}
+	if cluster.TLS != nil && cluster.SecurityProtocol != "SASL_SSL" {
+		saramaConfig.Net.TLS.Enable = true
+		tlsConfig := &tls.Config{
+			InsecureSkipVerify: cluster.TLS.Insecure,
+		}
+
+		caCert, _ := ioutil.ReadFile(cluster.TLS.Cafile)
```

Comment:
i would prefer if you check for err (don't just ignore the err). what about just trying to read the cacert if cluster.TLS.Cafile is != "" ? then, in case of no ca file path given, we won't have an error.

Reply:
sure thing. updated and tested:

```
ᐅ grep cafile ~/.kaf/config
    cafile: /tmp/foo
 ᐅ ./kaf node ls
Unable to read Cafile :open /tmp/foo: no such file or directory
```